StreetHawk/Classes/Notification/Internal/SHNotificationHandler.m

```diff
@@ -370,11 +370,14 @@ - (BOOL)handleDefinedUserInfo:(NSDictionary *)userInfo withAction:(SHNotificatio
                 }
                 if (!handledByCustomer)
                 {
-                    if (StreetHawk.developmentPlatform == SHDevelopmentPlatform_Native || StreetHawk.developmentPlatform == SHDevelopmentPlatform_Xamarin)
+                    SHDeepLinking *deepLinkingObj = [[SHDeepLinking alloc] init];
```

Comment:
nice .. 500 line function very good .. 

please do not make such big functions.